src/components/SetAndReSetPassword.vue

```diff
@@ -1,56 +1,114 @@
+
+      <v-text-field v-if="props.formData.text === 'Set Password' "
+       hide-details
+      density="compact"
+      class="inputShadow my-5 pt-2 pl-3 rounded"
+      color="info"
+      variant="plain"
+      name="email"
+      type="text"
+      :value="email"
+      :placeholder="email"
+      disabled
+      required />
+
+      <v-text-field v-if="props.formData.text === 'Set Password' "
+       hide-details
+      density="compact"
+      class="inputShadow my-5 pt-2 pl-3 rounded"
+      color="info"
+      variant="plain"
+      placeholder="Your Name"
+      name="name"
+      label="Name"
+      type="text"
+      v-model="data.name"
+      required />
+
+              <v-text-field  hide-details
+                density="compact"
+                class="inputShadow my-5 pt-2 pl-3 rounded"
+                color="info"
+                variant="plain"
+                placeholder="********"
                 name="newPassword"
                 label="New Password"
-                id="newPassword"
                 type="password"
                 v-model="data.newPassword"
                 required />
 
-              <v-text-field
-                name="newPasswordAgain"
-                label="New Password Again"
-                id="newPasswordAgain"
-                type="password"
-                v-model="data.newPasswordAgain"
-                required />
-            </v-col>
-            <v-col>
-              <v-btn color="primary" @click="$emit('buttonEvent',{token:route.query.token,...data})">{{props.formData.text}}</v-btn>
-                <button hidden @click.enter.prevent="$emit('buttonEvent',{token:route.query.token,...data})" />
-            </v-col>
-
-        </form>
-      </v-col>
-    </v-row >
-  </v-container>
+                <v-text-field  hide-details
+                  density="compact"
+                  class="inputShadow my-5 pt-2 pl-3 rounded"
+                  color="info"
+                  variant="plain"
+                  placeholder="********"
+                  name="newPasswordAgain"
+                  label="Confirm New Password"
+                  type="password"
+                  v-model="data.newPasswordAgain"
+                  required />
+
+                  <v-checkbox
+               label="info"
+               color="info"
+               value="I am human"
+               hide-details
+             ></v-checkbox>
+
+              <v-btn color="info" @click="$emit('buttonEvent',{token:route.query.token,...data})">{{props.formData.text}}</v-btn>
+              <button hidden @click.enter.prevent="$emit('buttonEvent',{token:route.query.token,...data})" />
+              <div>
+<p class="mt-4">We have sent you an e-mail with instructions on how to reset
+your password. Please check your inbox.</p>
+<v-btn color="white" class="mt-4" >continue</v-btn>
+</div>
+            </v-card-text>
+
+            </v-card>
+        </v-form>
 </template>
+
+<style scoped>
+
+.cardShadow {
```

Comment:
Please try to remove all of the custom css and replace them with built-in vuetify features if possible.